packages/client/tests/functional/init-error/tests.ts

```diff
@@ -0,0 +1,29 @@
+import { PrismaClientInitializationError } from '@prisma/engine-core'
+
+import testMatrix from './_matrix'
+
+// @ts-ignore this is just for type checks
+declare let prisma: import('@prisma/client').PrismaClient
+
+testMatrix.setupTestSuite(
+  () => {
+    // an example of how to query with the preloaded client
+    test('should assert that the error has the listed properties on the type', async () => {
+      expect.assertions(5)
+
+      try {
+        await prisma.$connect()
+      } catch (error) {
+        const e = error as PrismaClientInitializationError
+        expect(e.constructor.name).toEqual('PrismaClientInitializationError')
```

Comment:
If i use expect().instanceOf() I get a very strange error: 

```
expect(received).toBeInstanceOf(expected)

    Expected constructor: PrismaClientInitializationError
    Received constructor: PrismaClientInitializationError
```

This is the next best thing!

Reply:
It's become they don't come from the same module, the generated client for the tests is bundled.